checks/pull_requests.go

```diff
@@ -62,6 +62,8 @@ func PullRequests(c checker.Checker) checker.CheckResult {
 		}
 		if len(prs) > 0 {
 			totalWithPrs++
```

Comment:
i think we don't do Logf for two places where it is incremented, can you add a regular message for those too.

Reply:
Added a log here and a few lines up for gerrit logs.  PTAL